panther_lights/include/panther_lights/lights_driver_node.hpp

```diff
@@ -35,27 +35,28 @@ using ImageMsg = sensor_msgs::msg::Image;
 using SetBoolSrv = std_srvs::srv::SetBool;
 using SetLEDBrightnessSrv = panther_msgs::srv::SetLEDBrightness;
 
+enum LEDControlStatus {
+  IDLE = 0,
+  PENDING,
+  GRANTED,
+  NOT_GRANTED,
+  ERROR,
+};
```

Comment:
Not sure if it's necessary to maintain such a complex struct since we test `led_control_status_` only against `GRANTED` and `PENDING`

Reply:
There are actually 3 states that are tested: GRANTED, PENDING and (IDLE, NOT_GRANTED, ERROR - all together). I've added more states to be more descriptive and it could be useful for debugging or diagnostics later

Reply:
If we want to make use of these states in the current shape of the code, the only option is to add a `control_status` field on the diagnostics channel. (assuming that the newest logic on determining `OK` vs `ERROR` on diagnostics is changes)